app.rb

```diff
@@ -178,6 +178,27 @@ def get_alternate_region(provider, region)
     return regions
   end
 
+  def data_version_info
+    data_version = []
+    path_parts = []
+    return unless ENV['FRAMEWORKS']
+
+    if File.dirname(ENV['FRAMEWORKS']).include? 'pint-data'
+      path_parts = File.dirname(ENV['FRAMEWORKS']).partition('pint-data')
+    end
+    return if path_parts.empty?
+
+    path = path_parts[0].concat(path_parts[1]).concat('/.bumpversion.cfg')
```

Comment:
Nope, bumpversion.cfg is a file fo rthe bumpversion utilitiy it is strictly a development file and from another project. We need a different solution, we cannot depend on "internal" data.

I suggest to create a version.xml file or similar in the pint-data that then can be read by the server.

Reply:
bumpversion can update multiple destinations with the version, though, so it could write to XML as well as the specfile.